frontend/src/components/account/Register.jsx

```diff
@@ -275,183 +405,186 @@ opportunities from their partners. *
+            setMajor={setMajor} />
+          <FormHelperText style={{ fontSize: 9, color: "black", width: 300 }}>
+            * If your major is not in the list, choose 'other majors'
+          </FormHelperText>
+        </Grid>
+
+        <Grid item xs={12}>
+          <TextField
+            required variant="standard"
+            label="Program"
+            style={{ minWidth: 300, marginTop: -5.5, maxWidth: 300 }}
             onChange={(e) => {
               setGrad(e.target.value);
             }}
+            select
+            InputLabelProps={{ style: { color: '#000000' }, classes: { root: classes.label } }}
           >
             <MenuItem value="Undergraduate">Undergraduate</MenuItem>
             <MenuItem value="Graduate">Graduate</MenuItem>
             <MenuItem value="Postgraduate">Postgraduate</MenuItem>
-          </Select>
-        </FormControl>
+          </TextField>
+        </Grid>
+
+        <Grid container item xs={12} direction="row" alignItems="center" justify="center">
+          <Grid item xs={11} md={7}>
+            <TextField
+              required variant="standard"
+              label="Grad Month"
+              style={{ minWidth: 145, marginTop: 10, maxWidth: 145, marginBottom: -3 }}
+              onChange={(e) => {
+                setGrad_month(e.target.value);
+              }}
+              select
+              InputLabelProps={{ style: { color: '#000000' }, classes: { root: classes.label } }}
+            >
+              <MenuItem value="01">01</MenuItem>
+              <MenuItem value="02">02</MenuItem>
+              <MenuItem value="03">03</MenuItem>
+              <MenuItem value="04">04</MenuItem>
+              <MenuItem value="05">05</MenuItem>
+              <MenuItem value="06">06</MenuItem>
+              <MenuItem value="07">07</MenuItem>
+              <MenuItem value="08">08</MenuItem>
+              <MenuItem value="09">09</MenuItem>
+              <MenuItem value="10">10</MenuItem>
+              <MenuItem value="11">11</MenuItem>
+              <MenuItem value="12">12</MenuItem>
+            </TextField>
+          </Grid>
+
+          <Grid item xs={1} md={5}>
+            <TextField
+              required variant="standard"
+              label="Grad Year"
+              style={{ minWidth: 145, marginLeft: -12, marginTop: 10, maxWidth: 145, marginBottom: -3 }}
+              onChange={(e) => {
+                setGrad_year(e.target.value);
+              }}
+              select
+              InputLabelProps={{ style: { color: '#000000' }, classes: { root: classes.label } }}
+            >
+              <MenuItem value="2022">2022</MenuItem>
+              <MenuItem value="2023">2023</MenuItem>
+              <MenuItem value="2024">2024</MenuItem>
+              <MenuItem value="2025">2025</MenuItem>
+              <MenuItem value="2026">2026</MenuItem>
+              <MenuItem value="2027">2027</MenuItem>
+              <MenuItem value="2028">2028</MenuItem>
```

Comment:
Isn't there a month-year picker or date picker that would more elegantly allow users to choose what month and year they graduate?

Reply:
Good point, but it might not be consistent with the current dropdown style we have for the other user input fields.